ios/Classes/SwiftTwilioVoicePlugin.swift

```diff
@@ -865,12 +865,14 @@ public class SwiftTwilioVoicePlugin: NSObject, FlutterPlugin,  FlutterStreamHand
         }
     }
     
-    func reportIncomingCall(from: String, uuid: UUID) {
-        let callHandle = CXHandle(type: .generic, value: from)
+    func reportIncomingCall(from: String, uuid: UUID) {        
+        // Using .phoneNumber to fetch display name from the Phonebook if the number is saved otherwise show literal string 'from'.
+        let callHandle = CXHandle(type: .phoneNumber, value: from)  
```

Comment:
Great addition, though not all apps using twilio_voice make use of contacts or "from" values.